core/dbt/events/core_types_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: core_types.proto
-# Protobuf Python Version: 5.26.1
+# Protobuf Python Version: 4.25.3
```

Comment:
Let's ensure we're using 5.26.1 if possible to generate the protos: https://github.com/dbt-labs/dbt-core/pull/10456